drivers/flash/flash_stm32_ospi.c

```diff
@@ -2232,7 +2231,9 @@ static struct flash_stm32_ospi_data flash_stm32_ospi_dev_data = {
 		.Instance = (OCTOSPI_TypeDef *)DT_REG_ADDR(STM32_OSPI_NODE),
 		.Init = {
 			.FifoThreshold = STM32_OSPI_FIFO_THRESHOLD,
-			.SampleShifting = HAL_OSPI_SAMPLE_SHIFTING_NONE,
+			.SampleShifting = (DT_PROP(DT_PARENT(DT_DRV_INST(0)), ssht_enable)
```

Comment:
@erwango would be possible after #56815 merge that introduced that definition. I mentioned that in https://github.com/zephyrproject-rtos/zephyr/pull/56815#issuecomment-1523197339

Reply:
@erwango will be done in #56815 in a related commit that introduced that define. However, it requires this to be merged, then the other can be rebased and adapted.

Reply:
`STM32_OSPI_NODE` already exists, right ?

Reply:
@erwango it is added in https://github.com/zephyrproject-rtos/zephyr/pull/56815/commits/5b3aa32eca72f38d975b9878552c477ba90c691f that is a part of #56815 which is not merged.

Reply:
@GeorgeCGV I can see `#define STM32_OSPI_NODE DT_INST_PARENT(0)` in current main

Reply:
@erwango oh my, yes, probably it would be better to move it to the top. Will switch to using it here and move it up in the https://github.com/zephyrproject-rtos/zephyr/commit/5b3aa32eca72f38d975b9878552c477ba90c691f...